.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+.rnd
```

Comment:
Please remove this from the gitignore file.

Reply:
I'm using windows
![image](https://user-images.githubusercontent.com/79208489/231226788-a41277ac-0ddd-4d7f-ad65-6965b4e67637.png)